### PR TITLE
Add Nextra icon

### LINKED ISSUE #8492: Request new Nextra icon
### Brand Name

Nextra

### Website

https://nextra.site/

### Popularity Metric

The similar web rank is 1,093,997 and 6.6k stars on github. 

[https://github.com/shuding/nextra](https://github.com/shuding/nextra)

[https://www.similarweb.com/website/nextra.site/#overview](https://www.similarweb.com/website/nextra.site/#overview)

### Official Resources for Icon and Color

svg: https://nextra.site/about#design-assets
color: #000000

### Additional Comments

Nextra helps to generate the document and blog site. It is based on nextjs and maintained by vercel.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by officialrajdeepsingh**
**Issue:** closes #8492

**Similarweb rank:** 1,093,997

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Nextra has 6.6k stars on the github repo and also fixed the icon color issue as well.

### CODE REVIEW

**Comment by LitoMore** [comment]
CleanShot 2023-04-25 at 00 41 26@2x


Hi @officialrajdeepsingh, I see some differences.

It looks like you deleted the stroke from the source resource. We should keep it and convert it to outline, then combine it into one path.

The expected visual should be like the bottom one.

**Comment by officialrajdeepsingh** [comment]
Hii @LitoMore 
I found lots of errors related to width and size. I try to fix it using Figma, [Svg viewer](https://www.svgviewer.dev/), [Svg edit](https://aydos.com/svgedit/) and [svg path editor](https://yqnn.github.io/svg-path-editor/), etc. No one tools help me, i try many tools to fix my issue. Please help to recommend some tool which i used to fix dimension related error in my svg.

> x icons/nextra.svg
  x icon-size Size of <path> must be exactly 24 in one dimension; the size is currently 22.999 x 23
  x collinear-segments Collinear segment "-0.006 0.006" found in the chain "l-0.989 0.988 -0.006..." at index 482 (should be removed)
  x extraneous Unexpected newline character(s) detected in SVG markup
  x icon-centered <path> must be centered at (12, 12); the center is currently (11.996, 12.004)

**Comment by officialrajdeepsingh** [comment]
Hii @LitoMore 
I found lots of errors related to width and size. I try to fix it using Figma, [Svg viewer](https://www.svgviewer.dev/), [Svg edit](https://aydos.com/svgedit/) and [svg path editor](https://yqnn.github.io/svg-path-editor/), etc. No one tools help me, i try many tools to fix my issue. Please help to recommend some tool which i used to fix dimension related error in my svg.

> x icons/nextra.svg
  x icon-size Size of <path> must be exactly 24 in one dimension; the size is currently 22.999 x 23
  x collinear-segments Collinear segment "-0.006 0.006" found in the chain "l-0.989 0.988 -0.006..." at index 482 (should be removed)
  x extraneous Unexpected newline character(s) detected in SVG markup
  x icon-centered <path> must be centered at (12, 12); the center is currently (11.996, 12.004)

**Comment by LitoMore** [comment]
@officialrajdeepsingh Please follow this section and check your icon:

https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#4-annotate-the-icon

File: icons/nextra.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="m22.226.29-.989.988c-5.089 5.09-13.342 5.09-18.43 0L1.816.29A1.05 1.05 0 0 0 .335 1.774l.988.988c5.09 5.09 5.09 13.343 0 18.432l-.988.988a1.05 1.05 0 1 0 1.484 1.484l.988-.988c5.09-5.089 13.342-5.089 18.431 0l.989.988a1.05 1.05 0 0 0 1.484-1.484l-.988-.988c-5.09-5.09-5.09-13.343 0-18.432l.988-.988A1.05 1.05 0 0 0 22.226.29Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/nextra.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M23.677 22.111A1.109 1.109 0 0 1 22.894 24a1.1 1.1 0 0 1-.782-.324l-.983-.984A12.823 12.823 0 0 0 12 18.912a12.82 12.82 0 0 0-9.128 3.78l-.983.984a1.13 1.13 0 0 1-1.565 0 1.108 1.108 0 0 1 0-1.565l.983-.983c5.034-5.034 5.034-13.224 0-18.257l-.983-.983A1.099 1.099 0 0 1 0 1.106 1.107 1.107 0 0 1 1.888.324l.984.982c5.034 5.035 13.224 5.034 18.257 0l.983-.983a1.108 1.108 0 0 1 1.565 0 1.108 1.108 0 0 1 0 1.565l-.984.983c-5.033 5.033-5.033 13.223 0 18.257l.984.983z"/></svg>
```

This should fix the issues.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/nextra.svg (hunk 1 above)
Comment (PratyushJayachandran) [reply icons/nextra.svg:1]: Untitled
With fixed logo^

File: _data/simple-icons.json [hunk 2]
```diff
@@ -7708,6 +7708,12 @@
             "hex": "8ED500",
             "source": "https://about.nextdoor.com/us-media/"
         },
+        {
+            "title": "Nextra",
+            "hex": "000000",
+            "source": "https://nextra.site/",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:7714]:
```suggestion
            "source": "https://nextra.site",
```

Remove redundant trailing slash

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Remove unnecessary slash

File: icons/nextra.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M23.677 22.111A1.109 1.109 0 0 1 22.894 24a1.1 1.1 0 0 1-.782-.324l-.983-.984A12.823 12.823 0 0 0 12 18.912a12.82 12.82 0 0 0-9.128 3.78l-.983.984a1.13 1.13 0 0 1-1.565 0 1.108 1.108 0 0 1 0-1.565l.983-.983c5.034-5.034 5.034-13.224 0-18.257l-.983-.983A1.099 1.099 0 0 1 0 1.106 1.107 1.107 0 0 1 1.888.324l.984.982c5.034 5.035 13.224 5.034 18.257 0l.983-.983a1.108 1.108 0 0 1 1.565 0 1.108 1.108 0 0 1 0 1.565l-.984.983c-5.033 5.033-5.033 13.223 0 18.257l.984.983z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/nextra.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M22.604 20.422c-2.25-2.25-3.488-5.24-3.488-8.422s1.239-6.172 3.488-8.422l.945-.944a1.545 1.545 0 0 0 0-2.182 1.545 1.545 0 0 0-2.183 0l-.944.944c-4.644 4.644-12.2 4.644-16.843 0L2.635.452A1.533 1.533 0 0 0 1.543 0c-.412 0-.8.16-1.09.452A1.533 1.533 0 0 0 0 1.543c0 .412.16.8.452 1.091l.944.944c2.25 2.25 3.489 5.24 3.489 8.422s-1.24 6.172-3.489 8.422l-.944.944A1.534 1.534 0 0 0 0 22.456c0 .413.16.8.452 1.092.292.292.68.452 1.091.452.413 0 .8-.16 1.092-.452l.944-.944c4.644-4.643 12.2-4.643 16.843 0l.944.944c.602.602 1.58.601 2.183 0a1.545 1.545 0 0 0 0-2.182l-.945-.944z"/></svg>
```

I had to redo the SVG from source. But it should be fixed now.

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/658b3fae-0cef-42b7-838c-cb0db19e9688) when comparing the PR (in black) to the source icon (in red). Any chance you can have another look @officialrajdeepsingh?

File: icons/nextra.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M22.604 20.422c-2.25-2.25-3.488-5.24-3.488-8.422s1.239-6.172 3.488-8.422l.945-.944a1.545 1.545 0 0 0 0-2.182 1.545 1.545 0 0 0-2.183 0l-.944.944c-4.644 4.644-12.2 4.644-16.843 0L2.635.452A1.533 1.533 0 0 0 1.543 0c-.412 0-.8.16-1.09.452A1.533 1.533 0 0 0 0 1.543c0 .412.16.8.452 1.091l.944.944c2.25 2.25 3.489 5.24 3.489 8.422s-1.24 6.172-3.489 8.422l-.944.944A1.534 1.534 0 0 0 0 22.456c0 .413.16.8.452 1.092.292.292.68.452 1.091.452.413 0 .8-.16 1.092-.452l.944-.944c4.644-4.643 12.2-4.643 16.843 0l.944.944c.602.602 1.58.601 2.183 0a1.545 1.545 0 0 0 0-2.182l-.945-.944z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/nextra.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M22.68 21.031c-4.98-4.98-4.98-13.083 0-18.063l.978-.978c.22-.22.342-.513.342-.825 0-.311-.122-.604-.342-.824-.44-.441-1.207-.44-1.648 0l-.979.978c-4.98 4.98-13.084 4.98-18.063 0L1.99.34a1.17 1.17 0 0 0-1.649 0 1.168 1.168 0 0 0 0 1.649l.978.978c4.98 4.98 4.98 13.083 0 18.063l-.977.978c-.221.22-.342.513-.342.825 0 .31.121.604.341.824.442.443 1.21.441 1.65 0l.977-.977c4.98-4.983 13.083-4.98 18.064 0l.978.977c.22.22.513.342.824.342.312 0 .605-.122.824-.342.22-.22.342-.512.342-.824 0-.313-.122-.605-.342-.825l-.977-.978z"/></svg>
```

I outlined the stroke first before scaling it to 24px. Please merge this. I will explain the issue in a separate comment.
nextra

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by PratyushJayachandran** [comment]
> I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/658b3fae-0cef-42b7-838c-cb0db19e9688) when comparing the PR (in black) to the source icon (in red). Any chance you can have another look @officialrajdeepsingh?

The issue is happening because the original icon has a stroke around it. It needs to be outlined and then combined together before scaling it up to 24px. I might have outlined it after scaling, making it go out of proportion with original. Please make sure to do the same when comparing with source.

CC: @LitoMore

**Review by LitoMore — APPROVED** [review]
image

Looks good. Thanks.

**Pull request merged**